.vscode/tasks.json

```diff
@@ -75,6 +75,22 @@
         "panel": "new"
       },
       "problemMatcher": []
+    },
+    {
+      "label": "Build current project",
+      "type": "shell",
+      "command": "pwsh",
+      "args": [
+        "${workspaceFolder}/eng/scripts/vscode-build.ps1",
```

Comment:
Can we get a bash variant for this so that this task works in a Codespaces environment?

Reply:
I did this PR in codespaces only. And interestingly it worked there. Seems pwsh (PowerShell) is pre-installed there!

Reply:
This makes me worry a bit but I'm more concerned about developers building directly on non-Windows platforms. Can't assume `pwsh` will be available for everyone, even if the Codespaces environment includes it.

Reply:
yeah, I can understand! I will try to add a bash variant as well!